Source/Flow/Private/Nodes/Route/FlowNode_CustomInput.cpp

```diff
@@ -1,17 +1,14 @@
 // Copyright https://github.com/MothCocoon/FlowGraph/graphs/contributors
 
 #include "Nodes/Route/FlowNode_CustomInput.h"
+#include "FlowSettings.h"
+
+#define LOCTEXT_NAMESPACE "FlowNode"
 
 UFlowNode_CustomInput::UFlowNode_CustomInput(const FObjectInitializer& ObjectInitializer)
 	: Super(ObjectInitializer)
 {
```

Comment:
_Most of the CustomInput/Output code moved into CustomNodeBase, so this diff doesn't look so great (mostly removed a bunch of common stuff)_